server/api/standardized_contests.py

```diff
@@ -30,7 +30,7 @@
 def process_standardized_contests_file(
     session: Session, election: Election, file: File
 ):
-    def process():
+    def process() -> None:
```

Comment:
realized this wasn't getting typechecked because it didn't have any type annotation. changes below are just for the typechecker